src/router/index.js

```diff
@@ -57,6 +58,11 @@ const router = createRouter({
       name: "curation-entries",
       component: ListCurationView,
     },
+    {
+      path: "/lgd/add",
```

Comment:
I think the draft url should be /curate or something else, we are not adding or creating a LGD yet